pages/agent/v3/installation.md.erb

```diff
@@ -0,0 +1,34 @@
+# How to Install Buildkite Agent (v3)
+
+The Buildkite Agent runs on your own machine, whether it's a VPS, server, desktop computer, embedded device. There are installers for:
+
+* [Ubuntu](ubuntu)
+* [Debian](debian)
+* [CentOS/Redhat](redhat)
+* [FreeBSD](freebsd)
+* [macOS](osx)
+* [Windows](windows)
+* [Linux](linux)
+* [Docker](docker)
+
+Alternatively you can install it manually using the instructions below.
+
+## Manual Installation
+
+If you need to install the agent on a system not listed above you'll need to perform a manual installation using one of the binaries from [buildkite-agent’s releases page](https://github.com/buildkite/agent/releases).
+
+Once you have a binary you simply create a bin and builds directory in `~/.buildkite-agent` and copy the binary and `bootstrap.sh` file into place:
```

Comment:
Random question: 
@lox is this still the case with the bootstrap file? We have a question in the issues here (bk/docs) about where the bootstrap file comes from, so I was having a poke around and saw something about the shell version of bootstrap being removed from v3?